driver-core/src/test/java/com/datastax/driver/core/EventDebouncerTest.java

```diff
@@ -65,6 +67,33 @@ public void should_deliver_single_event() throws InterruptedException {
     }
 
     @Test(groups = "unit")
+    public void should_log_and_drop_events_on_overflow() throws InterruptedException {
```

Comment:
When executed alone it takes < 300ms (a lot of that could be overhead in the framework though).  Makes sense to lower the max though just to verify the behavior in tests though :+1: